### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by CreaCloudz** [hexle-at/DiscordChatExporter-JDA#1]
Are you going to add custom emojis in DiscordChatExporter ?

**Comment by Pianist20005** [comment]
If you want, we can probably add this feature to the chat exporter!

**Comment by CreaCloudz** [comment]
that would be good because I'm using JDA 5 and suddenly the rendering of emojis doesn't work at all

**Comment by Pianist20005** [comment]
Yeah i'll have a closer look when i'm at home and inform you when we think this is added, so it works fine!

**Comment by CreaCloudz** [comment]
No problem thanks

**Comment by CreaCloudz** [comment]
You have already seen these HTML files they are really perfect. 
https://github.com/Tyrrrz/DiscordChatExporter

**Comment by Pianist20005** [comment]
I will be more available in the next days - besides of that i just had a look onto the chatexporter!
We are going to add more features in the next weeks! Different export settings and more render types.
